Automattic/jetpack
# SEO: standardize sanitization for seo inputs

#### Changes proposed in this Pull Request:

- `sanitize_front_page_meta_description` will use `wp_strip_all_tags`
- `sanitize_title_formats` will use code from `wp_strip_all_tags`, but
we need to preserve the spacing (e.g. no trim)

Fixes #19414

#### Jetpack product discussion

* Internal: p8oabR-DS-p2#comment-5185

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* After applying changes, navigate to `/wp-admin/admin.php?page=jetpack#/traffic`, and down to the front page meta description section.
* Try saving a string like: `test <script>alert('hi');</script>` After saving the `<script>` portion should be removed.
* The same should be repeated for a custom page title structure.
